README.md

```diff
@@ -1,17 +1,33 @@
 # Package Support Framework
-This project provides tools, libraries, documentation and samples for creating app-compat fixups to enable classic Win32 applications to be distributed and executed as packaged apps.
+This project provides tools, libraries, documentation and samples for creating app-compat fixups to enable classic Win32 applications to be distributed and executed as MSIX packaged apps.
+
+Here are some common examples where you can find the PSF useful:
+	1. Your app can't find some .dlls when launched, you may need to set your current working directory.
+	You can learn about the required current working directory in the original shortcut before you converted to MSIX.
+	2. The app writes into the install folder. You will typically see it by "Access Denied" errors in 
+	[Process Monitor] (https://docs.microsoft.com/en-us/sysinternals/downloads/procmon) 
+	3. Your app needs to pass parameters to the executable on launch.
+	You can learn more about how to identify compatability issues [here] (https://docs.microsoft.com/en-us/windows/msix/psf/package-support-framework#identify-packaged-application-compatibility-issues)
+	And about the available configurations [here] or in the Documention section below (https://github.com/microsoft/MSIX-PackageSupportFramework/tree/master/PsfLauncher)
+
+We are looking for your feedback, please file any issues or feedback in the [Issues section] (https://github.com/Microsoft/MSIX-PackageSupportFramework/issues) or in our [tech community] (https://techcommunity.microsoft.com/t5/Package-Support-Framework/bd-p/Package-Support)
 
 ## Documentation
-Check out our [step by step guide](https://docs.microsoft.com/en-us/windows/uwp/porting/package-support-framework), it will walk you through the main PSF workflows and provides the key documentation.
+Check out our [step by step guide](https://docs.microsoft.com/en-us/windows/uwp/porting/package-support-framework), it will walk you through the main PSF workflows and provides the key documentation. Check out the [PSF scripts support] (https://docs.microsoft.com/en-us/windows/msix/psf/run-scripts-with-package-support-framework) that we recently added to our framework, it provides great flexibility to yuour packages.
 
 See also:
 * [Package Support Framework package layout](layout.md)
 * [Package Support Framework Nuget package install](https://www.nuget.org/packages/Microsoft.PackageSupportFramework)
 * [Instructions for authoring your own shim dll](Authoring.md)

```

Comment:
>shim [](start = 39, length = 4)

fixup

Reply:
Should use term fixup

---
In reply to: [337138171](https://github.com/microsoft/MSIX-PackageSupportFramework/pull/96#discussion_r337138171) [](ancestors = 337138171)